### DISCUSSION TITLE
Restart R?

### DISCUSSION THREAD

**Discussion by joelnitta** [REditorSupport/vscode-R#897] [Q&A]
Is there a way to restart R from within an active terminal (equivalent of RStudio "Session" -> "Restart R")?

I didn't see anything here https://github.com/REditorSupport/vscode-R/wiki/Interacting-with-R-terminals

**Comment by eitsupi** [comment]
This post may be useful.
https://github.com/randy3k/radian/issues/296#issuecomment-974583380
